lib/cloud_controller/deployment_updater/actions/cleanup_web_processes.rb

```diff
@@ -0,0 +1,21 @@
+module VCAP::CloudController
+  module DeploymentUpdater
+    module Actions
+      class CleanupWebProcesses
```

Comment:
**thought** I wonder if it would be cleaner if `CleanupWebProcesses`/`ScaleDownOldProcess`/`ScaleDownSuperceded` could be one class whose responsibly is to handle the scaling/destruction of processes. `ProcessScaler` or something